dev_scripts/find_images_to_rebuild.py

```diff
@@ -29,7 +36,8 @@
     print("Usage: ./lightblue.py ERRATA_ID [[CONTAINER_IMAGE], ...]")
     sys.exit(1)
 
-container_images = sys.argv[2:]
+container_images = [container_image for container_image in sys.argv[2:] if
```

Comment:
Add an argument in argsparse instead to avoid guessing what the container images. This approach is very fragile.